core/modules/system/css/system.theme.css

```diff
@@ -585,6 +585,19 @@ tr:hover del.diffchange {
 }
 
 /**
+ * Hero blocks.
+ */
+ .hero-block {
+   background-size: cover;
+   background-repeat: no-repeat;
+   background-position: top center;
+   padding: 50px 100px;
+   text-align: center;
+   color: #ffffff;
```

Comment:
I think we either have to add config around font color, or leave it as is.

Leaning towards the latter.